aiidalab_widgets_base/computational_resources.py

```diff
@@ -425,8 +425,15 @@ def _is_in_config(self):
         fpath = Path.home() / ".ssh" / "config"
         if not fpath.exists():
             return False
-        cfglines = open(fpath).read().split("\n")
-        return "Host " + self.hostname.value in cfglines
+        sshcfg = parse_sshconfig(self.hostname.value)
+        # Confusingly, parse_sshconfig returns a dict with a hostname
+        # even if it is not in the config file.
+        if len(sshcfg.keys()) < 2:
+            return False
```

Comment:
I believe this condition is redundant. The hostname will be always present in the dictionary (even if the host is not present in the `.ssh/config` file. The other required key is the username, which is checked in the next line. So I would omit this check entirely.

Reply:
Yeah, I thought about this as well. I've removed it.